docs/guide/features.md

```diff
@@ -239,7 +239,7 @@ All test and coverage results can be merged at the end of your CI pipeline using
 ```bash
 vitest --shard=1/2 --reporter=blob
 vitest --shard=2/2 --reporter=blob
```

Comment:
We should probably have it here too.

```suggestion
vitest --shard=1/2 --reporter=blob --coverage
vitest --shard=2/2 --reporter=blob --coverage
```

Reply:
Accepting the suggestion added a commit that didn't follow the standards, I squashed it into the original commit